src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -130,11 +137,30 @@ trait SharedFileSystem {
       WdlFile(WdlFile.appendPathsWithSlashSeparators("/root", pathWithWorkflowName.subpath(1, pathWithWorkflowName.getNameCount).toString))
     }
 
-    // If this call is using Docker, adjust input paths, otherwise return unaltered input paths.
+    def localizeInputToCallDirectory(path: String): Path = {
+      val originalPath = Paths.get(path)
+      val callPath = Try {
+        val callInputsDirectory = LocalBackend.hostCallPath(workflowDescriptor, call.name, callKey.index).resolve("inputs")
+        if (!callInputsDirectory.toFile.exists) callInputsDirectory.toFile.mkdirs
+        val hash = hashPath(originalPath)
+        callInputsDirectory.resolve(s"$hash-${originalPath.getFileName.toString}")
+      }
+
+      callPath.toOption flatMap { cp =>
+        CallLocalizers map { _(Option(callKey.scope), originalPath, cp) } find { _.isSuccess } map { _ => cp }
+      } getOrElse {
+        logger.warn(s"Unable to isolate call inputs for call ${call.name}")
```

Comment:
Thinking more about it it probably should yes. The thing is for workflow-level inputs it doesn't really have to be since the files have already been localized successfully in workflow-inputs (we wouldn't have started the workflow if localization failed). So we could still default to point to the originalPath (which is what is done here).

But for inputs coming from other calls outputs, it's possible that those other calls failed and didn't produce the expected output, which we will then fail to localize. This makes us fail earlier as the call won't even start in this case.